Config.xml

```diff
@@ -51,5 +52,6 @@
     <File>CustomizedMapping.lua</File>
     <File>WebClient.lua</File>
     <File>AlmaApi.lua</File>
+    <File>OnFormClosing.elf</File>
```

Comment:
I'm not seeing this file in the repo. Was it supposed to be added? I noticed it being used in Catalog.lua